utils/results/zip_intermediate.py

```diff
@@ -68,41 +67,51 @@ def zip_selected(root_dir, dir_name, output_filename, selected_files, selected_d
         if sel not in dirs_found:
             log.warning("Looked for %s but could not find it.", sel)
 
-    os.chdir(cwd)  # Get back to where you once belonged
+    os.chdir(FWV0)  # Get back to where you once belonged
 
 
-def zip_intermediate_selected(context, run_label):
+def zip_intermediate_selected(
+    gear_intermediate_files,
+    gear_intermediate_folders,
+    destination_id,
+    gear_name,
+    output_dir,
+    work_dir,
+    run_label,
+):
     """Zip the listed files and folders in work/.
 
     Args:
-        context (gear_toolkit.GearToolkitContext): flywheel gear context
+        gear_intermediate_files (str) space separated list of files
+        gear_intermediate_folders (str) space separated list of folders
+        destination_id (str) ID of analysis container that is the destination of the gear
+        gear_name (str) name of gear from manifest "name"
+        output_dir (str) path to where output will be written
+        work_dir (str) path to temporary directory
         run_label (str) name of run to use in zip file name
     """
 
     do_find = False
     files = []
     folders = []
     # get list of intermediate files (if any)
-    if context.config.get("gear-intermediate-files", None):
-        if len(context.config["gear-intermediate-files"]) > 0:
-            files = context.config["gear-intermediate-files"].split()
+    if gear_intermediate_files:
```

Comment:
hmmm...Since `gear_intermediate_files` and `gear_intermediate_folders` are strings, I think this check is redundant.  Python typecasts empty strings to "False" when in a boolean, even if they've been declared.  

```
In [35]: a=""                                                                                                                              

In [36]: len(a)                                                                                                                            
Out[36]: 0

In [37]: if a: 
    ...:     print(True) 
    ...: else: 
    ...:     print('False') 
    ...:                                                                                                                                   
False
```

Same below.

Reply:
Good catch!